Sources/OpenPass/OpenPassManager.swift

```diff
@@ -298,6 +298,18 @@ public final class OpenPassManager {
         }
     }
 
+    public var deviceAuthorizationFlow: DeviceAuthorizationFlow {
```

Comment:
Thank you for not using `DAF` in the production code or API. I think it's fine in the Dev App, but we already have multiple different names we are calling this across SDKs, introducing "DAF" as another one would trigger me 😅